app/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,83 @@
+<resources>
+    <string name="app_name">Mis gastos</string>
```

Comment:
Correct me if I'm wrong, but doesn't that mean "My expenses"? Would "Gastos" not be enough?

Reply:
You are right, I also have the doubt with  welcome_to_expenses because then the translation should be "Bienvenido a Gastos" which sound weird to me.
We can also maintain the name of the app without translation 🤔

Reply:
Does "Gastos" sounds weird to you? Regarding `welcome_to_expenses`, you can use the same trick as I did with Polish translation, something like "Bienvenido a la aplicación Gastos".

Reply:
Gastos as name is fine c: